optimism/src/mips/interpreter.rs

```diff
@@ -34,9 +38,9 @@ pub enum RTypeInstruction {
     SyscallWriteOther,            // syscall (Write ?)
     SyscallFcntl,                 // syscall (Fcntl)
     SyscallOther,                 // syscall (Brk, Clone, ?)
-    MoveZero,                     // movz
-    MoveNonZero,                  // movn
-    Sync,                         // sync
+    MoveZero,                     // movz - FIXME: pseudo?
+    MoveNonZero,                  // movn - FIXME: pseudo?
+    Sync,                         // sync - FIXME: pseudo?
```

Comment:
`sync` is documented in its own right in the manual. Doesn't look "pseudo" to me